docs/dunst.pod

```diff
@@ -264,6 +264,15 @@ The progress value can be set with a hint, too.
 
 =back
 
+=head1 ACTIONS
+
+Dunst allows interaction with actions in notifications. If "show_indicators" is
```

Comment:
This should probably also mention that actions are not user-configurable and that the program that sent the notification handles them.
Additionally, this sentence is weirdly structured, I think 
> Dunst allows programs to specify actions that the user can choose to perform.

should cover it with minimal confusion.
Maybe call 'programs' notifiers? Not sure.